src/controllers/userController.js

```diff
@@ -168,6 +177,20 @@ const getCurrentUserWithPurchasedAlbums = async (req, res) => {
   */
 };
 
+const updateProfileImage = async (req, res) => {
+  console.log('Updating profile image.');
+  const firebaseStorage = getStorage(firebaseApp);
+  const user = req.user;
+  const imageRef = ref(firebaseStorage, `images/${user._id}_profile_pic`);
```

Comment:
When `authenticateUser` middleware initializes the `req.user` variable, it puts the id in a `userId` property rather than `_id` : [https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/9f81dd657a81ecb96b06946e251aaae8cf26152f/src/middleware/authentication.js#L16](https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/9f81dd657a81ecb96b06946e251aaae8cf26152f/src/middleware/authentication.js#L16)

```suggestion
  const imageRef = ref(firebaseStorage, `images/${user.userId}_profile_pic`);
```